README.md

```diff
@@ -8,4 +8,5 @@ The `ARGO_*` variables are the same ones as show under the user info tab.
 * `ARGO_SERVER` the name of the of the server (optionally a port)
 * `ARGO_TOKEN` the secret token to use to authenticate
 * `ARGO_NAMESPACE` in which namspace the events should be posted
-* `GERRIT_SERVER` is of the form `<username>@<host>:<port>`. It will use the standard ssh keys.
+* `GERRIT_SERVER` is of the form `<username>@<host>:<port>`. It will use the standard ssh keys, unless `SSH_PRIVATE_KEY` is set.
+* `SSH_PRIVATE_KEY` is the optional openssh private in text form.
```

Comment:
typo: missing the word "key" after private.

Is it really the KEY or the path to a file containing the key?

Reply:
It was, I've changed it to a file and adopted the variable name and description.